src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEAutomatic.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const COEAutomatic = () => (
+  <>
+    <va-alert status="success">
+      <h3 slot="headline">Congratulations on your automatic COE</h3>
+      <div>
+        <p>
+          We have all the information we need, so you don’t need to fill out an
+          application. You can download your COE now.
+        </p>
+      </div>
+    </va-alert>
+    <div>
+      <h2>Review and download your COE</h2>
+      <p>
+        You can download your COE right now. If you need help, go to our
+        instructions for how to download and open a VA.gov PDF form.
+      </p>
+      <a href="#">Get instructions for downloading a VA.gov PDF</a>
+    </div>
+    <div className="vads-u-padding-top--4">
+      <a href="#">
+        <i className="fas fa-download vads-u-padding-right--1" /> Download your
+        COE (PDF) 0.20MB
+      </a>
+    </div>
+    <div>
+      <h2>What if my COE has errors?</h2>
+      <p>What if I need to make changes to my COE?</p>
+      <ul>
+        <li>
+          Make changes to your COE (correct an error or update your
+          information), <strong>or</strong>
+        </li>
+        <li>Apply for a restoration of entitlement</li>
+      </ul>
+    </div>
+  </>
```

Comment:
I think this section needs to be updated according to the signed-in auto coe [mockup](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/141257590/simulate/sitemap). Looks like that sentence sub text needs to be added.

<img width="734" alt="Screen Shot 2021-09-17 at 10 22 41 AM" src="https://user-images.githubusercontent.com/15097156/133808037-29361481-08ec-4665-8a63-de38ecf0da56.png">

Reply:
Derp, updated